src/Validation/Collector/EmptyConfigurableRectorCollector.php

```diff
@@ -14,6 +15,18 @@
  */
 final class EmptyConfigurableRectorCollector
 {
+    /**
+     * These are rector rules that the default values config are not empty array
+     *
+     * @var array<class-string<RectorInterface>>
+     */
+    private const ALLOWED_RULES_FALLBACK_DEFAULT_CONFIG = [
```

Comment:
would it make sense if these rectors would implement a marker-interface instead of having a hardcoded list in this class?

Reply:
I will try to re-apply original PR functionality to check value of config property https://github.com/rectorphp/rector-src/pull/960 , see my latest comment https://github.com/rectorphp/rector-src/pull/1414#issuecomment-987968347